src/ot_orchestration/utils/labels.py

```diff
@@ -0,0 +1,57 @@
+"""Labels for resources in Google Cloud."""
+
+import re
+
+from ot_orchestration.utils.common import GCP_PROJECT_PLATFORM, shared_labels
+
+
+class Labels:
+    """A collection of labels for Google Cloud resources.
+
+    Includes a set of default labels, and ensures that all labels are correctly
+    formatted.
+
+    Refer to the `controlled vocabularies <https://github.com/opentargets/controlled-vocabularies/blob/main/infrastructure.yaml>`__
```

Comment:
So the controlled vocab does not contain `genetics` team 🥶

Reply:
AFAIK it's extremely work in progress. I only saw it used in one place before, but I think it's a good idea. You should open a PR for it, I already added some stuff to it myself. :D